eng/common/scripts/apiview-helper.ps1

```diff
@@ -0,0 +1,56 @@
+
```

Comment:
nit: Empty lines

Please name the file "ApiView-Helpers.ps1" and move it under the Helpers folder.